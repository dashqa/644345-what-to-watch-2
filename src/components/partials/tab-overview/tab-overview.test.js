import React from 'react';
import renderer from 'react-test-renderer';
import TabOverview from './tab-overview';

const props = {
  rating: 4.4,
  scoresCount: 100500,
  description: `My description`,
  director: `Lynne Ramsay`,
  starring: [`Tilda Swinton`, `John C. Reilly`],
};

it(`Overview tab correctly renders after relaunch`, () => {
  const tree = renderer.create(<TabOverview {...props}/>).toJSON();
  expect(tree).toMatchSnapshot();
});
