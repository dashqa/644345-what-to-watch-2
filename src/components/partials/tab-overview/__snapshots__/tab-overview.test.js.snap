// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Overview tab correctly renders after relaunch 1`] = `
Array [
  <div
    className="movie-rating"
  >
    <div
      className="movie-rating__score"
    >
      4.4
    </div>
    <p
      className="movie-rating__meta"
    >
      <span
        className="movie-rating__level"
      >
        Normal
      </span>
      <span
        className="movie-rating__count"
      >
        100500
         ratings
      </span>
    </p>
  </div>,
  <div
    className="movie-card__text"
  >
    <p>
      My description
    </p>
    <p
      className="movie-card__director"
    >
      <strong>
        Director: 
        Lynne Ramsay
      </strong>
    </p>
    <p
      className="movie-card__starring"
    >
      <strong>
        Starring: 
        Tilda Swinton, John C. Reilly
      </strong>
    </p>
  </div>,
]
`;
